### ISSUE TITLE
Long description text not showing completely

### ISSUE THREAD

**Issue by stubshqappdeveloper** [Spikeysanju/MotionToast#56]
Hi, there is issue in description text of toast. It shows only text in 2 lines and longer text not showing completely.

I noticed that you have set `android:maxLines="2"` to `android:id="@+id/custom_toast_description"` TextView in motion_toast.xml file.

Can you please make height of TextView open or increase maxLines.

Thanks

**Comment by ResulAkkiz** [comment]
Yeah ,same issue for me. It's important and neccessary. I am suprised that they didn't solve or think about it while developing
